spyse/client.py

```diff
@@ -62,34 +62,35 @@ def __init__(self, results: List[WHOISHistoricalRecord], total_items: int = None
         self.search_id: Optional[str] = search_id
         self.results: List[WHOISHistoricalRecord] = results
 
-
+from datetime import datetime
 class Client:
     DEFAULT_BASE_URL = 'https://api.spyse.com/v4/data'
     MAX_LIMIT = 100
-    SEARCH_RESULTS_LIMIT = 10000
-    RATE_LIMIT_FRAME_IN_SECONDS = 1
+    CAPACITY = 1
+    RATE_LIMIT_FRAME_IN_SECONDS = 0.85
```

Comment:
This solves the `spyse.response.RateLimitError: too many requests` issue for me with a Community plan. It could make sense to adjust the value during init for the other plans,